### PR TITLE
123 - Fixing too strict version constraints

### PR DESCRIPTION
**Pull request by vansari**

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
     "require-dev": {
         "roave/security-advisories": "dev-latest",
         "codeception/codeception": "4.1.*",
```
Comment (Naktibalda) [inline composer.json:31]: How about supporting codeception/codeception 4.2?

File: composer.json [hunk 2]
```diff
@@ -29,11 +29,11 @@
...
-        "squizlabs/php_codesniffer": "^3.6.0",
-        "overtrue/phplint": "^3.0.0",
-        "friendsofphp/php-cs-fixer": "^3.0.0",
+        "squizlabs/php_codesniffer": "^3.6",
+        "overtrue/phplint": "^3.0",
+        "friendsofphp/php-cs-fixer": "^3.0",
         "phpunit/phpunit": ">=9.0",
```
Comment (Naktibalda) [inline composer.json:35]: PHPUnit 10 is not supported, so this constraint should be `^9.5`.

File: composer.json (hunk 1 above)
Comment (vansari) [reply composer.json:31]: Hi @Naktibalda you're right! I will change it.

File: composer.json (hunk 2 above)
Comment (vansari) [reply composer.json:35]: Ah Thanks!

**Review by Naktibalda — APPROVED** [review]

**Pull request merged**